src/main/webapp/WEB-INF/views/series/info.html

```diff
@@ -623,6 +623,8 @@ <h5 th:text="#{t_who_selling_series}">Who was selling/buying this series</h5>
 								</div>
 							</div>
 							
+							<div id="series-sales-list" sec:authorize="hasAuthority('VIEW_SERIES_SALES')"></div>
```

Comment:
We need to render this `<div>` only when react feature toggle is enabled:
```suggestion
							<div id="series-sales-list" sec:authorize="hasAuthority('VIEW_SERIES_SALES')" togglz:active="USE_REACT"></div>
```

In order to make it work, the following should be added to the top of the page:

https://github.com/php-coder/mystamps/blob/3ebd680ecfdfdeca497fe38ff2871afe86f3bfdb/src/main/webapp/WEB-INF/views/site/index.html#L6

Also, we need to disable rendering of the current code when we use React, so, the line 562 should be modified:
```diff
-<div class="row" th:if="${not #lists.isEmpty(seriesSales)}" sec:authorize="hasAuthority('VIEW_SERIES_SALES')">
+<div class="row" th:if="${not #lists.isEmpty(seriesSales)}" sec:authorize="hasAuthority('VIEW_SERIES_SALES')" togglz:inactive="USE_REACT">
```